VTUL/iawa
# LIBTD-1790: Remove display of list of files in Item show page

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-1790) (:star:)

* Other Relevant Links (Meeting note, project page, related pull requests, etc.)

# What does this Pull Request do? (:star:)
This PR removes the display of image files in Item show page. For end users, displaying the images in Mirador viewer is the key function, as the number of files gets large, adding the list would bring confusions.

# What's the changes? (:star:)

* Remove the list of image files in Item display page
* Remove Batch import button from the interface as it will be performed through Rails task

# How should this be tested?

* See nothing below the Item metadata in Item show page
* The "Batch Import" button disappears from the dashboard --> works page

# Additional Notes:

* What branch to be used for testing this PR? LIBTD-1790

# Interested parties
Tag (@yinlinchen ) interested parties

(:star:) Required fields
